src/components/search/History/SearchHistory.scss

```diff
@@ -0,0 +1,37 @@
+@import "src/styles/scaffold";
+
+.history__title {
+  align-self: center;
+  font-size: var(--text-size-3);
+  line-height: 1.5;
+}
+
+.history__header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.history__section {
+  padding: var(--space-1) var(--space-3) var(--space-2);
+  background-color: var(--bg-neutral-lightest);
+}
+
+[data-store-search-history] {
+  ol {
+    margin-top: var(--space-1);
+  }
+
+  li {
+    font-size: var(--text-size-2);
+
+    [data-store-link] {
+      display: flex;
+      justify-content: space-between;
+      color: var(--color-neutral-6);
+    }
+
+    [data-store-icon] {
+      margin-right: var(--space-1);
+    }
```

Comment:
We need to align the rows. Like the first row in the image below:
<img width="294" alt="Screen Shot 2022-03-16 at 19 59 49" src="https://user-images.githubusercontent.com/11325562/158705362-bb8f379b-b441-423e-84cc-4966a5896169.png">

So, we need to tweak it a little.
(also considering that the `margin-right` just need to apply on the first icon. If we need to apply to both icons, just ignore the `&:first-child`).
```suggestion
   [data-store-link] {
      display: flex;
      align-items: center;
      justify-content: space-between;
      color: var(--color-neutral-6);

      div {
        display: flex;
        align-items: center;
      }
    }

    [data-store-icon] {
      display: flex;

      &:first-child { margin-right: var(--space-1); }
    }
```